pkg/storage/mysql/mysqlstore_test.go

```diff
@@ -331,13 +339,107 @@ func TestSQLDBStore(t *testing.T) {
 		require.NoError(t, err)
 		verifyItr(t, itr, 1, "")
 
-		itr, err = store.Query(`""`)
-		require.Error(t, err)
+		itr, e := store.Query(`""`)
+		require.Error(t, e)
 		require.Nil(t, itr)
-		require.Contains(t, err.Error(), "failed to query rows")
+		require.Contains(t, e.Error(), "failed to query rows")
```

Comment:
Is there a reason for renaming `err` to `e`?

Reply:
linter was complaining shadow err